src/interceptors/XMLHttpRequest/XMLHttpRequestOverride.ts

```diff
@@ -84,7 +86,7 @@ export const createXMLHttpRequestOverride = (
     public statusText: string
     public user?: string
     public password?: string
-    public data: string
+    public data: ArrayBuffer
```

Comment:
Can we make this assumption that XHR `data` will always be buffer?

Note that this `data` represents whichever payload was fed to XHR upon construction. Could you please share some more context as to why you're making this change?

Reply:
Removed